Makefile

```
@@ -507,20 +506,20 @@ clear-stalled:
 	fi
 
 .PHONY: test-build
-test-build: | all build-addons build-js-native-api-tests build-node-api-tests
+test-build: | all build-addons build-js-native-api-tests build-node-api-tests ## Build all tests.
 
 .PHONY: test-build-js-native-api
-test-build-js-native-api: all build-js-native-api-tests
+test-build-js-native-api: all build-js-native-api-tests ## Build JS Native-API tests.
 
 .PHONY: test-build-node-api
-test-build-node-api: all build-node-api-tests
+test-build-node-api: all build-node-api-tests ## Build Node-API tests.
 
 .PHONY: test-all
 test-all: test-build ## Run default tests with both Debug and Release builds.
 	$(PYTHON) tools/test.py $(PARALLEL_ARGS) --mode=debug,release
 
 .PHONY: test-all-valgrind
-test-all-valgrind: test-build
+test-all-valgrind: test-build ## Test all valgrind tests
```

Most comments have a final dot, some do not. Can we make that consistent?
```suggestion
test-all-valgrind: test-build ## Test all valgrind tests.
```

---

I don't understand what it means to test valgrind tests.

---

I'm honestly not 100% sure, but I believe it's <https://valgrind.org/docs/manual/quick-start.html>

---

Ahh, I've updated the wording